cluster/gce/manifests/etcd.manifest

```diff
@@ -14,20 +14,15 @@
 "priorityClassName": "system-node-critical",
 "priority": 2000001000,
 "hostNetwork": true,
-"containers":[
+"initContainers":[
     {
-    "name": "etcd-container",
+    "name": "etcd{{ suffix }}-migrate-container",
     "image": "{{ pillar.get('etcd_docker_repository', 'k8s.gcr.io/etcd') }}:{{ pillar.get('etcd_docker_tag', '3.4.13-0') }}",
-    "resources": {
-      "requests": {
-        "cpu": {{ cpulimit }}
-      }
-    },
     "command": [
-              "/bin/sh",
-              "-c",
-              "set -o errexit; if [ -e /usr/local/bin/migrate-if-needed.sh ]; then /usr/local/bin/migrate-if-needed.sh 1>>/var/log/etcd{{ suffix }}.log 2>&1; fi; exec /usr/local/bin/etcd --name etcd-{{ hostname }} --listen-peer-urls {{ etcd_protocol }}://{{ host_ip }}:{{ server_port }} --initial-advertise-peer-urls {{ etcd_protocol }}://{{ hostname }}:{{ server_port }} --advertise-client-urls {{ etcd_apiserver_protocol }}://127.0.0.1:{{ port }} --listen-client-urls {{ etcd_apiserver_protocol }}://{{ listen_client_ip }}:{{ port }} {{ quota_bytes }} --data-dir /var/etcd/data{{ suffix }} --initial-cluster-state {{ cluster_state }} --initial-cluster {{ etcd_cluster }} {{ etcd_creds }} {{ etcd_apiserver_creds }} {{ etcd_extra_args }} 1>>/var/log/etcd{{ suffix }}.log 2>&1"
-            ],
+        "/bin/sh",
+	"-c",
+        "set -o errexit; if [ -e /usr/local/bin/migrate-if-needed.sh ]; then /usr/local/bin/migrate-if-needed.sh 1>>/var/log/etcd{{ suffix }}.log 2>&1; fi;"
```

Comment:
same question about what happens if this fails - retries? terminates?

Reply:
It will retry. See [this](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#understanding-init-containers) for details.